### ISSUE TITLE
Module not found: Error: Can't resolve 'deepmerge'

### ISSUE THREAD

**Issue by RikdeVos** [denniske/ngx-translate-multi-http-loader#10]
Hi,

I've updated Angular from v12 to v13. This package was working fine. But after updating "ngx-translate-multi-http-loader" from 3.0.0 to 7.0.0 I get following build the error:
```
./node_modules/ngx-translate-multi-http-loader/fesm2020/ngx-translate-multi-http-loader.mjs:3:0-30 - Error: Module not found: Error: Can't resolve 'deepmerge' in '...node_modules/ngx-translate-multi-http-loader/fesm2020'
```
Any ideas?

I am using
* "@ngx-translate/core": "14.0.0"
* Angular 13.0.3
* Node 14.17.3
* NPM 6.14.13
* MacOS

**Comment by denniske** [comment]
Hi, I just published 7.0.1 where I moved deepmerge from peer dependencies to normal dependencies. Please check if that solves your issue.

Just some background why I had it in peer dependencies initially:
[https://stackoverflow.com/questions/51216616/what-is-the-proper-way-to-use-a-dependent-npm-package-in-angular-6-library-proje](https://stackoverflow.com/questions/51216616/what-is-the-proper-way-to-use-a-dependent-npm-package-in-angular-6-library-proje)

**Comment by RikdeVos** [comment]
Hi @denniske

I've tested it and works perfectly now in my project, thanks for your prompt response!
